### ISSUE TITLE
Please Add Support for the Spesmilo Currency Symbol (U+20B7)

### ISSUE THREAD

**Issue by gitjeff2** [alerque/libertinus#336]
After seeing #329, I decided to check Libertinus support for lesser-known Esperanto glyphs, and it seems one is missing: the [*Spesmilo*](https://en.wikipedia.org/wiki/Spesmilo) currency symbol.

Prior to the [Great War](https://en.wikipedia.org/wiki/World_War_I) there was a unit of currency associated with the Esperanto movement called the Spesmilo. The privately minted currency was gold backed and intended to be an international unit of exchange, each Spesmilo being worth 0.733g of gold. The Spesmilo symbol appeared on coinage and later on banknotes. After the war, the Spesmilo was eclipsed by the [Stelo](https://en.wikipedia.org/wiki/Stelo) which just uses the Black Star (U+9733) or White Star (U+U9734) Unicode symbols in the absence of a dedicated currency glyph.

The Spesmilo glyph was [assigned U+20B7 in Unicode 5.2](http://www.unicode.org/charts//PDF/Unicode-5.2/U52-20A0.pdf) (PDF). While not especially common today, the symbol is starting to see *increased* use. The [original petition to the Unicode Consortium](http://unicode.org/wg2/docs/n3390.pdf) (PDF) cites the Esperanto language edition of Monopoly as a common use case. Newer and more serious use-cases I could cite are:

* [*La Esperanto-Numismatika Asocio*](http://ena.frali.bplaced.net/) (The Esperanto Numismatic Association) which has a number of documents dealing with coinage directly or indirectly associated with Esperanto.
* [*La Esperantomuzeo kaj Kolekto por Planlingvoj*](https://en.wikipedia.org/wiki/Esperanto_Museum_and_Collection_of_Planned_Languages) (Esperanto Museum and Collection of Planned Languages) has been steadily digitizing their collection, some of it about the Spesmilo and historical international currency issues.
* Practically speaking, there is nothing stopping someone from privately minting a coin containing 0.733g of gold in the future and slapping a Spesmilo on it &mdash; or doing basically the same thing with a blockchian based [smart contract](https://en.wikipedia.org/wiki/Smart_contract) digitally.

Would it be possible to add this glyph to Libertinus?

**Comment by alerque** [comment]
I'd be happy to accept a contribution if somebody drew a glyph that fit the design of the font(s). I'm like to see both Serif and Sans supported in contributions for new coverage where possible.

I don't exactly have the machinery spun up yet to review and accept contributions (most importantly testing for regressions), but the existing build and normalization stuff is pretty close. If somebody set out to add a glyph and opened a PR I'm sure we can extract the right bits when I'm ready.

**Comment by gitjeff2** [comment]
@alerque, I can barely find my way around Font Forge as I'm still early in the learning curve. Someone may well beat me to the punch, but I'll see if I can make a reference image in Inscape that more or less follows Libertinus' design language. Will that work as a starting point?

**Comment by alerque** [comment]
Yes, a reference image would help. I assume we'll use elements of the S & m characters from the respective fonts to start with and figure out the joins from there.

Also any other fonts (particularly ones with licenses that we could borrow from) that implement this would be nice to know about.

**Comment by waldyrious** [comment]
@gitjeff2 I sympathize with your situation, as I recently tried FontForge for the first time to contribute some small changes to this repo. I found that FontForge is actually pretty similar to Inkscape in many aspects, so if you're familiar with the latter, maybe some of my [notes to self](https://gist.github.com/waldyrious/e55a284379a9e09f9df5f677924ef1a6) may help you jump through some of the hoops more easily :)

At the time, as I was learning the ropes of using FontForge and contributing to the Libertinus project, I also submitted some changes to Libertinus' README and [contribution documentation](https://github.com/alerque/libertinus/blob/main/CONTRIBUTING.md) to make it more beginner-friendly; I encourage you to also add any details you feel would have helped you once you get past the hurdles. The small stuff is easy to (dis)miss after you're used to the tools, but make a considerable difference in allowing new contributors to get involved, as I'm sure you'll appreciate.
